src/plugins/intel_cpu/src/utils/debug_caps_config.cpp

```diff
@@ -66,6 +66,12 @@ void DebugCapsConfig::readProperties() {
 
     if ((envVarValue = readEnv("OV_CPU_AVERAGE_COUNTERS")))
         averageCountersPath = envVarValue;
+
+    if ((envVarValue = readEnv("OV_CPU_MEMORY_STATISTICS_LEVEL")))
+        memoryStatisticsDumpLevel = std::stoi(envVarValue);
```

Comment:
I think we better to use some default path, like `ov_cpu_memory_profile.csv`, when level is specified, to avoid always requiring a use of two environment variables to make the feature work.

Reply:
Done